src/routes/index.ts

```diff
@@ -32,7 +32,12 @@ export default async function (fastify: FastifyInstance): Promise<void> {
 
   // Debugging endpoint
   fastify.post('/e', (req, res) => {
-    req.log.info({ body: req.body }, 'events received');
+    req.log.debug({ body: req.body }, 'events received');
+    return res.status(204).send();
+  });
+
+  fastify.post('/e/x', (req, res) => {
+    req.log.debug({ body: req.body }, 'allocation received');
```

Comment:
This should post `api.experiment-allocated` topic in the future?

Reply:
Nope, it's just a debugging endpoint for local development. The real endpoint is in analytics-api